core/src/test/resources/log4j.properties

```diff
@@ -14,5 +14,8 @@
 # limitations under the License.
 ##
 
-log4j.rootLogger=OFF
+log4j.rootLogger=INFO, stdout
```

Comment:
I used it for debugging, yes. However, I thought enabling logging in general would be a good thing. I'm happy to change it back if you disagree.